src/main/rx/cc2500_frsky_d.c

```diff
@@ -217,12 +218,12 @@ rx_spi_received_e frSkyDHandlePacket(uint8_t * const packet, uint8_t * const pro
                     if (packet[0] == 0x11) {
                         if ((packet[1] == rxFrSkySpiConfig()->bindTxId[0]) &&
                             (packet[2] == rxFrSkySpiConfig()->bindTxId[1])) {
-                            LedOn();
+                            cc2500LedOn();
                             nextChannel(1);
+                            cc2500setRssiDbm(packet[18]);
```

Comment:
Not sure if this is right (the underlying FrSky D protocol has only been reverse engineered), we probably should still check `(packet[3] % 4) == 2)` before setting this.

Reply:
No, that should be fine then.